src/board/mod.rs

```diff
@@ -32,14 +32,14 @@ use ruleset::Ruleset;
 use score::Score;
 use self::point::Point;
```

Comment:
Yes, sorry. No coffee and commenting on an iPhone during a commute are my excuses. ;)